### DISCUSSION TITLE
CSS pre-processing docs?

### DISCUSSION THREAD

**Discussion by alxbradley** [thheller/shadow-cljs#1095] [Q&A]
How do you configure css pre-processing? I can't find any documentation on how this works although I can see "sass" and "scss" referenced in the code.

Can we add documentation for this?

**Comment by thheller** [comment]
You can't find anything because there isn't anything to find. `shadow-cljs` has no support for processing CSS in any way. There are some old remnants of experiments I did, which however never turned into anything useful.

In general my stance is that CSS processing should be done by separate tools, running separately. shadow-cljs will [reload css](https://shadow-cljs.github.io/docs/UsersGuide.html#_css_reloading) when configured, but that is only taking the final `.css` output file, it is not involved in creating it in any way.
